tests/passes/cell-share/inline.futil

```diff
@@ -0,0 +1,31 @@
+// -p pre-opt -p post-opt
```

Comment:
Thanks for adding the test! Can you add a line explaining what should happen with this test? Sharing behavior changes over time so comment-level documentation is helpful for these tests.